client/src/components/sign-up.js

```diff
@@ -1,98 +1,108 @@
-import React, { Component } from 'react'
-import axios from 'axios'
+import React, { Component } from "react";
+import axios from "axios";
 
 class Signup extends Component {
-	constructor() {
-		super()
-		this.state = {
-			email: '',
-			password: '',
-			confirmPassword: '',
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      confirmPassword: "",
+      flashMessage: ""
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+  handleSubmit(event) {
+    event.preventDefault();
+    const { email, password } = this.state;
+    //request to server to add a new username/password
+    axios
+      .post("/user", {
```

Comment:
this is probably hard to read having this post on a separate line. Keep it axios.post(...)